lib/kuzzle/kuzzle.js

```diff
@@ -42,11 +40,16 @@ const Logger = require('./log');
 const vault = require('./vault');
 const DumpGenerator = require('./dumpGenerator');
 const AsyncStore = require('../util/asyncStore');
+const InternalIndexHandler = require('./internalIndexHandler');
+const Funnel = require('../api/funnel');
+const EntryPoint = require('../core/network/entryPoint');
 const CacheEngine = require('../core/cache/cacheEngine');
 const StorageEngine = require('../core/storage/storageEngine');
 const SecurityModule = require('../core/security');
 const RealtimeModule = require('../core/realtime');
-const InternalIndexHandler = require('./internalIndexHandler');
+
+// kuzzle instance singleton
+global.kuzzle = null;
```

Comment:
@Aschen > this is now obsolete. Could you merge #1913's branch on this one, and mark this PR as dependent of it?